docs/library/network.WLAN.rst

```diff
@@ -125,10 +125,10 @@ Methods
    Parameter      Description
    =============  ===========
    mac            MAC address (bytes)
-   essid          WiFi access point name (string)
+   ssid           WiFi access point name (string)
    channel        WiFi channel (integer)
-   hidden         Whether ESSID is hidden (boolean)
-   authmode       Authentication mode supported (enumeration, see module constants)
+   hidden         Whether SSID is hidden (boolean)
+   security       Security protocol supported (enumeration, see module constants)
    password       Access password (string)
```

Comment:
I think this needs changing to "key"

Reply:
Replaced all the related `password` in the docs.

Reply:
Note ESP ports use positional `ssid` and `key` not keyword args, but I fixed them anyway in their docs/quickref/tutorials.